index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/0bd1156d30.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="Main-Page">
+     <nav class="nav">
+<div class="logo">AFIAMOAH</div>
+
+<div class="nav-links-menu"><p class="side-logo"><i class="fa-solid fa-bars"></i></p></div>
```

Comment:
- [ ] The hamburger menu on the other hand is specifically used for navigation, so kindly replace the `<div class="nav-links-menu">` with a `<nav class="nav-links-menu">` to improve the semantic structure of the website.